cmd/credentialUtil.go

```diff
@@ -562,8 +562,6 @@ func GetCredentialInfoForLocation(ctx context.Context, location common.Location,
 func getCredentialType(ctx context.Context, raw rawFromToInfo, cpkOptions common.CpkOptions) (credType common.CredentialType, err error) {
 
 	switch {
-	case raw.fromTo == common.EFromTo.S3Blob(): // S3 Bucket
-		credType, _, err = getCredentialTypeForLocation(ctx, raw.fromTo.From(), raw.source, raw.sourceSAS, true, common.CpkOptions{})
```

Comment:
Was this code added in 10.12?
If not, was this scenario working in 10.11?

Reply:
Yes, it was added in 10.12! 

I also tested 10.11 and this scenario was working.